tests/Type/ResolveInfoTest.php

```diff
@@ -648,4 +648,263 @@ public function testPathAndUnaliasedPathForList(): void
             ],
         ], $result);
     }
+
+    private int $aliasArgsNbTests = 0;
```

Comment:
Use a local variable that is included in the closure by reference (`use (&$var)`).